decidim-forms/app/views/decidim/forms/questionnaires/answers/_matrix_multiple.html.erb

```diff
@@ -2,7 +2,7 @@
   <table>
     <thead>
       <tr>
-        <th></th>
+        <td></td>
```

Comment:
Why not use `colspan` on the first heading instead of leaving an empty cell?

Reply:
Because it's easier by far. Otherwise you'll need more complicated css rules to accomplish it.

Current:
![image](https://github.com/decidim/decidim/assets/817526/851f0688-9df0-41db-b09d-b0348dfed09a)
Colspan:
![image](https://github.com/decidim/decidim/assets/817526/50cdfb51-7b3b-4523-b4d5-5f566d51d914)